apparmor.d/profiles-s-z/zsys-system-autosnapshot

```diff
@@ -0,0 +1,34 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2022 Jeroen Rijken
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}{s,}bin/zsys-system-autosnapshot
+profile zsys-system-autosnapshot @{exec_path} flags=(complain) {
+  include <abstractions/base>
+
+  @{exec_path}           rm,
+  /{usr/,}bin/{,ba,da}sh rix,
+  /{usr/,}bin/cat        rix,
+  /{usr/,}bin/cp         rix,
+  /{usr/,}bin/rm         rix,
+  /{usr/,}bin/zsysctl    rix,
+  /{usr/,}bin/zsysd      rix,
+
+  /{usr/,}lib/locale/locale-archive r,
```

Comment:
already included in the base abstraction, so remove it here.

Reply:
The locale folder itself yes, but not individual files within it:
```
$ grep "locale" -R ./abstractions/
./abstractions/base.d/complete:  /usr/share/locale/             r,
./abstractions/totem:  owner @{user_cache_dirs}/tracker/db-locale.txt r,
```

Reply:
It is part of the default base abstraction, see: https://gitlab.com/apparmor/apparmor/-/blob/b65fbf5fbd916da8bed3dbc72932192d09641df5/profiles/apparmor.d/abstractions/base#L50

Reply:
Ah, that one.